airdrop/airdrop-distributor/tests/functions/end_block.rs

```diff
@@ -0,0 +1,31 @@
+use crate::utils::{
+    airdrop_distributor_abi_calls::{airdrop_constructor, end_block},
+    test_helpers::{defaults, setup},
+};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn returns_end_block() {
+        let (deploy_wallet, wallet1, wallet2, wallet3, asset) = setup().await;
+        let (_, _, _, _, _, _, _, _, claim_time) =
+            defaults(&deploy_wallet, &wallet1, &wallet2, &wallet3).await;
+        let provider = deploy_wallet.wallet.get_provider().unwrap();
+        let root = [2u8; 32];
+
+        airdrop_constructor(
+            asset.asset_id,
+            claim_time,
+            &deploy_wallet.airdrop_distributor,
+            root,
+        )
+        .await;
+
+        assert_eq!(
+            end_block(&deploy_wallet.airdrop_distributor).await,
+            provider.latest_block_height().await.unwrap() + claim_time - 1
```

Comment:
Is there a reason for the `-1`?

Reply:
I've enabled manual blocks [here](https://github.com/FuelLabs/sway-applications/blob/6cc956d75d19e57e23bbe719a1ffd328296f3840/airdrop/airdrop-distributor/tests/utils/mod.rs#L266). Every time a call is made the block height increases. So calling the `airdrop_constructor` would assign the block height at say 1. Then `end_block` would be 2. Getting the `latest_block_height()` would then need to be 1 less as we have moved forward 1 block in time from when we assigned the claim time in the constructor. 

I've gone ahead and flipped these so that we do not need to subtract by one.